src/components/pages/StatesPage/States.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import State from './State';
+import WeatherInfo from '../WeatherPage/WeatherInfo';
+
+const States = () => {
+  const location = useLocation();
+  const countryName = location.state.name;
+  const countryState = useSelector((state) => (state.countries[countryName]));
+  return (
+    <div className="statesPage" data-testid="statesContent">
+      <div className="weatherInfo">
+        <WeatherInfo
+          name={countryName}
+        />
+      </div>
+      <div className="padding chooseState">
+        <h2>
+          States
+        </h2>
+        <p>
+          Choose a state:
+          {' '}
+          {countryState.states.length}
+          {' '}
+          States
+        </p>
+      </div>
+      <div className="states padding">
+        {countryState.states.map((state) => (
+          <State
+            name={state.name}
+            key={state.name}
+            code={state.state_code}
+          />
+        ))}
+      </div>
+    </div>
```

Comment:
- [ ] Kindly stick to the original design as much as possible. Please make sure that the list of states in each country is displayed one in each row, to match the original design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/177816545-be47d645-289c-4475-a9ef-1c5efb3877b2.png) | ![image](https://user-images.githubusercontent.com/62290545/177816634-fe4db3aa-f707-4cb5-9dc6-8b80d03772de.png) |

Reply:
Well resolved 👍🏽